README.md

```diff
@@ -49,14 +57,16 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
   </form>
 ```
 
+var user_input = $('#color').append($'#Submit');
```

Comment:
You've selected the right ID - `$('color')`. It doesn't make sense to append that to anything. Instead, you need to use the .val() method to get what the user typed. So `var user_input = $('#color').val()`.